fs/test/fs_zenfs_test.cc

```diff
@@ -0,0 +1,83 @@
+
+
+#include "../fs_zenfs.h"
+
+#include <gtest/gtest.h>
+
+#include <cstdlib>
+#include <filesystem>
+#include <memory>
+
+#include "../util.h"
+#include "rocksdb/convenience.h"
+#include "rocksdb/env.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+class FsZenFSTest : public testing::Test {
+ private:
+  std::filesystem::path aux_path;
+
+  void uniqueue_path(std::filesystem::path& out_path) {
+    char* name = std::tmpnam(nullptr);
```

Comment:
Is there a specific reason to use `tmpnam`? When compiling the test case I am getting the following warning: 
```
warning: the use of `tmpnam' is dangerous, better use `mkstemp'`
```

Reply:
Probably we want to ignore that warning because we just want to return a unique path that is then created by ZenFS's mkfs.

Reply:
@MaisenbacherD  I am OK with the risk in this test case.